website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -775,6 +817,94 @@ spec:
 ```
 {{% /alert %}}
 
+### AL2023
+
+* Your UserData may be in one of three formats: a [MIME multi part archive](https://cloudinit.readthedocs.io/en/latest/topics/format.html#mime-multi-part-archive), a NodeConfig YAML / JSON string, or a shell script.
```

Comment:
nit: If we are making a list here, should this list be offset and numbered or bulleted?

Reply:
This was another case of following the precedent set by the other AMI families, if I were writing it in a vacuum it probably wouldn't be a list at all.